src/symbolic-node.jl

```diff
@@ -220,6 +220,51 @@ function get_name(self :: mx.SymbolicNode)
     return Symbol(unsafe_string(name[]))
 end
 
+import Base: print
+
+function print(io :: IO, sym :: SymbolicNode)
+  out = Ref{mx.char_p}(C_NULL)
+  @mx.mxcall(:MXSymbolPrint, (mx.MX_SymbolHandle, Ref{mx.char_p}), sym.handle, out)
```

Comment:
Who is responsible for deallocating the string?

Reply:
https://github.com/apache/incubator-mxnet/blob/470b56437290b33fef51b067c96fdce08cefa584/include/mxnet/c_api.h#L933

```
MXNET_DLL int MXSymbolPrint(SymbolHandle symbol, const char **out_str);
```

(I'm not skilled at C) 
Do I need to free the content of `const char**`?

Reply:
I think technically we are taking  ownership of this ptr so yeah we need to free it after we are done with it, (probably elsewhere as well.)